src/main/java/org/zone/region/flag/entity/player/move/message/greetings/GreetingsFlagType.java

```diff
@@ -30,12 +34,25 @@
 
     @Override
     public @NotNull GreetingsFlag load(@NotNull ConfigurationNode node) throws IOException {
-        String message = node.node("Message").getString();
+        String message = node.node("GreetingsMessage").getString();
+        String displayType = node.node("GreetingsMessageDisplayType").getString();
```

Comment:
The root node that is provided tells the user what flag it is in, therefore the prefix of ``GreetingsMessage`` can be removed

Reply:
oh much better